config/jobs/image-pushing/k8s-staging-e2e-test-images.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+
+readonly OUTPUT="$(dirname $0)/k8s-staging-e2e-test-images.yaml"
+readonly IMAGES=(
+    agnhost
+    apparmor-loader
+    busybox
+    cuda-vector-add
+    echoserver
+    ipc-utils
+    jessie-dnsutils
+    kitten
+    metadata-concealment
+    nautilus
+    node-perf
+    nonewprivs
+    nonroot
+    pets
+    redis
+    regression-issue-74839
+    resource-consumer
+    sample-apiserver
+    sample-device-plugin
+    volume
+    windows
```

Comment:
In here we have Dockerfiles for a few Windows helper images that are supposed to be built on Windows nodes, something that we don't have in GCB. They aren't meant to be rebuilt, or rebuilt very rarely. You should exclude this.

Reply:
done